src/models/contracts/languageService.ts

```diff
@@ -111,3 +111,27 @@ export class DidChangeLanguageFlavorParams {
 export namespace LanguageFlavorChangedNotification {
     export const type = new NotificationType<DidChangeLanguageFlavorParams, void>('connection/languageflavorchanged');
 }
+
+
+// ------------------------------- < Load Completion Extension Request > ------------------------------------
+/**
+ * Completion extension load parameters
+ */
+export class CompletionExtensionParams {
+    /// <summary>
+    /// Absolute path for the assembly containing the completion extension
+    /// </summary>
+    public assemblyPath: string;
+    /// <summary>
+    /// The type name for the completion extension
+    /// </summary>
+    public typeName: string;
+    /// <summary>
+    /// Property bag for initializing the completion extension
+    /// </summary>
+    public properties: {};
```

Comment:
we don't have any better type here?

Reply:
This is to match the properties dictionary needed for initializing a CompletionExtension.